test/testdata/argslen.go

```diff
@@ -0,0 +1,7 @@
+//args: -Eargslen
+//config: linters-settings.argslen.maxArguments=1
+package testdata
+
+func argslenFunc(s1, s2, s3 string) { // ERROR "args number for function `argslenFunc` is too high, 3 vs 1 allowed"
```

Comment:
Can the error be reworded (just to make it more clear)?
1) args number is reached the allowed maximum of 1 argument?
2) args number for function `argslenFunc` is too high, found 3 of 1 allowed

or maybe someone can come with even more clear statement.

Reply:
I agree with @butuzov about rephrasing this and I think "args number (for function)" can be hard to understand for some people. If we talk about too many arguments I think that should be the error message. Something like.

> Too many arguments for function `argslenFunc`, 1 allowed but got 3